ldaptive/src/test/resources/org/ccci/idm/user/ldaptive/dao/ldap.xml

```diff
@@ -15,9 +15,7 @@
       <bean class="org.ldaptive.ConnectionConfig"
           p:ldapUrl="${ldap.url}"
           p:useSSL="true"
-          p:sslConfig-ref="ldap.config.ssl"
-          p:connectTimeout="${ldap.connectTimeout:5000}"
-          p:responseTimeout="${ldap.responseTimeout:5000}">
```

Comment:
I'm relying on defaults. to keep the config here for tests would involve a bunch of spring magic since it now takes a java 8 duration object instead of an int.